### ISSUE TITLE
waveform is always partially transparent

### ISSUE THREAD

**Issue by Janneman84** [dmrschmidt/DSWaveformImage#20]
I'm trying to create a black waveform but it turns dark grey. I tried making it white too using `style: .filled(.white)` but the yellow background still seeps through making it look light yellow:

<img width="223" alt="Schermafbeelding 2021-02-10 om 11 22 31" src="https://user-images.githubusercontent.com/9085167/107497053-562b2f00-6b92-11eb-9133-e0713e87fd09.png">

I took a look at the drawing code but couldn't find why this happens. I sort of fixed it by just adding an extra stroke:

```
case .filled, .striped:
    context.setStrokeColor(configuration.color.cgColor)
    context.strokePath()
    context.addPath(path)
    context.strokePath()
```

The waveform is now much more black/white this way, but I wonder why this hack is necessary.

<img width="225" alt="Schermafbeelding 2021-02-10 om 11 31 32" src="https://user-images.githubusercontent.com/9085167/107498103-a35bd080-6b93-11eb-9faf-237cc7b6d94e.png">

I've had this issue on 6.1.1 and now on 7.0.0 too.

**Comment by dmrschmidt** [comment]
Yeah this has been a weird one to me for a while now. I wanted to spend some time again today afternoon to look into it again. My best guess right now is that it has something to do with antialiasing since it only happens in the filled case, which is essentially drawing a bunch of 1px lines.

A more "native workaround" that doesn't require any code changes to be used - since 7.0.0 - would be to use `.gradient([yourColor, yourColor])`. So a gradient with the same color twice.

**Comment by dmrschmidt** [comment]
It was indeed antialiasing. This is fixed - and configurable - now in 7.0.1.

**Comment by Janneman84** [comment]
I just tried and even with antialiasing the waveform is fully opaque now, thanks!
